web-client/public/js/update-app.js

```diff
@@ -447,14 +483,7 @@ const resetDatasetDropdownMenus = (network) => {
     }
 
     // Add expression database options
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Barreto_2018_wt"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Dahlquist_2018_dcin5"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Dahlquist_2018_dgln3"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Dahlquist_2018_dhap4"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Dahlquist_2018_dzap1"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Dahlquist_2018_wt"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Kitagawa_2002_wt"});
-    grnState.nodeColoring.nodeColoringOptions.push({value: "Thorsen_2007_wt"});
+    expressionDBDatasets.forEach(option => grnState.nodeColoring.nodeColoringOptions.push({value: [option]}));
```

Comment:
Nice change! Makes things more data-driven. That way, additions to `expressionDBDatasets` in the future will automatically expand the list of values.